JS/Lesson26/saffet/createPost.js

```diff
@@ -0,0 +1,46 @@
+const container = document.getElementById("container");
+const postArea = document.getElementById("postArea");
+const form = document.getElementById("postForm");
+const titleInput = document.getElementById("title");
+const bodyInput = document.getElementById("body");
+const saveButton = document.getElementById("saveButton");
+const successMessage = document.getElementById("successMessage");
+const urlParams = new URLSearchParams(window.location.search);
+const postId = urlParams.get("id");
+
+saveButton.addEventListener("click", (event) => {
+
+    event.preventDefault();
+
+    if(titleInput.value.trim() === "" || bodyInput.value.trim() === ""){
+        alert("Please fill out both title and body fields");
```

Comment:
Do you remember a lesson where we did a form validation? Try to add error messages to input fields in that way instead of showing an alert to the use.